pkg/instance/handler.go

```diff
@@ -569,31 +570,23 @@ func (h Handler) Delete(c *gin.Context) {
 		return
 	}
 
-	token, err := handler.GetTokenFromHttpAuthHeader(c)
-	if err != nil {
-		_ = c.Error(err)
-		return
-	}
-
 	instance, err := h.instanceService.FindById(uint(id))
```

Comment:
Aren't these already returning a NotFound error? In that case we can simplify the if (and similar ones in any handler) to

```go
	instance, err := h.instanceService.FindById(uint(id))
    if err != nil {
	    _ = c.Error(err)
	    return
    }
```